src/Umbraco.Web.UI.NetCore/appsettings.json

```diff
@@ -24,6 +24,10 @@
         "DisableKeepAliveTask": false,
         "KeepAlivePingUrl": "{umbracoApplicationUrl}/api/keepalive/ping"
       },
+      "ModelsBuilder": {
+        "ModelsMode": "Nothing",
```

Comment:
I agree this should be nothing as default value

Reply:
Just to make sure it's clear here, I had to change this as I found out that currently in `netcore/netcore` we aren't loading the configuration in appSettings (as it's looking for it in the wrong place).  When I fixed that, and `PureLive` was getting picked up, I was getting errors so changed the default to `Nothing`.

Reply:
Is `Nothing` one of the enum values? Not sold on the naming here or the need for separate values.

`None` or `Disabled` would be a better choice.

Reply:
It is yes.  Like the example below, we'd need to consider whether changing the name here that might be in various people's solution config is worthwhile.

Reply:
I very much doubt there’ll be an upgrade path from V8 that allows preserving exisiting config

Reply:
The reason you had errors is most likely because PureLive and Precompiled views are not a good combi :)  So if you use PureLive you will also have to disable the precompiled views.

Reply:
_I very much doubt there’ll be an upgrade path from V8 that allows preserving exisiting config_ - no, agreed.  I was more meaning though that if someone has this config in the existing XML, then it'll be a copy/paste process to transfer it into JSON.  If the names change, then it's a little bit more.